tests/utils/k8s.go

```diff
@@ -66,9 +69,64 @@ func DropNamespace(clientSet *kubernetes.Clientset, name string) error {
 	})
 }
 
+func getPodLog(clientSet *kubernetes.Clientset, namespace string, pod string, lines int64) (string, error) {
```

Comment:
I think `lines` is a bit confusing name for a variable meant for `tail` operation, can we rename it to something like `tailLines`?